docs/quickstarts/outlook-quickstart.md

```diff
@@ -79,11 +79,22 @@ The add-in project that you've created with the Yeoman generator contains sample
 1. In your code editor, open the file **./src/taskpane/taskpane.js** and add the following code within the **run** function. This code uses the Office JavaScript API to get a reference to the current message and write its **subject** property value to the task pane.
```

Comment:
Thanks for submitting this PR, @bonfix. Because the `Office.onReady` code is already included in the **taskpane.js** file when you create your add-in project using the Yeoman generator, we didn't include it in this step. Instead, we highlighted the JavaScript code to include in the `run` function.

Let me know if this clarifies this step for you or if there's opportunity for us to revise this step.

Reply:
Yes, that clarifies it, I had missed the part that says inside the run function

Reply:
Though it might work better for beginners if the code included the full code for the file including Office.onReady

Reply:
Thanks, @bonfix. I incorporated your feedback into this PR. I retained the current guidance we have now and added the entire contents of the **taskpane.js** file for clarity. If you're happy with these changes, please follow the instructions in [Incorporating feedback in your pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request) to accept the suggestions.